### ISSUE TITLE
Red line sticks to graph

### ISSUE THREAD

**Issue by abuhurairalakdawala** [vigoss6666/stocker#1]
![untitled](https://user-images.githubusercontent.com/6679615/52518815-00f07600-2c77-11e9-8933-cb4f6678a208.png)

the hovering line sticks to the graph on circle points...

**Comment by zaid-zaheer-shaikh** [comment]
Yes, hovering and crossair lines needs to work together. Minor problem
